### PR TITLE
#5 0:00–0:15

### PR DESCRIPTION
**Pull request by jsphm**

### CODE REVIEW

File: 5/jamie-white-captions.srt [hunk 1]
```diff
@@ -220,58 +221,59 @@ there's quite a few of them.
...
 
 53
 00:04:08,925 --> 00:04:10,137
-I got it off Github.
+I cloned it off Github...
 
 54
 00:04:11,300 --> 00:04:13,460
-With npm, what you tend to do
+With NPM, what you tend to do
```
Comment (nelstrom) [inline 5/jamie-white-captions.srt]: I don't think I've seen NPM capitalized like that before. I thought it was always all lowercase. Even the company "npm, Inc." uses lowercase, so I think that's probably the way to go.

**Comment by nelstrom** [comment]
Nice work @jsphm, thanks!

File: 5/jamie-white-captions.srt [hunk 2]
```diff
@@ -531,11 +532,12 @@ Let's say for the beginning...
 
 126
 00:09:40,350 --> 00:09:42,770
-display a list of episodes.
+Let's just display a display
+a list of episodes.
```
Comment (nelstrom) [inline 5/jamie-white-captions.srt:536]: This looks like a mistake: "display a display a list"

File: 5/jamie-white-captions.srt [hunk 3]
```diff
@@ -649,24 +655,24 @@ because it looks the most like...
 
 154
 00:12:17,840 --> 00:12:22,502
-I don't have application roots yet,
-the roots of where data gets loaded in,
+I don't have an applcation route yet.
```
Comment (jsphm) [inline 5/jamie-white-captions.srt:658]: Are you happy with this one @nelstrom? 'application route' could also be ApplicationRoute. The Ember guides used the latter until the introduction of Ember CLI. Perhaps either is fine.

Comment (nelstrom) [inline 5/jamie-white-captions.srt:658]: Let's stick with 'application route'.

Comment (jsphm) [inline 5/jamie-white-captions.srt:658]: :+1:

**Pull request merged**